cookbook/bundles/dependencies.rst

```diff
@@ -0,0 +1,51 @@
+.. index::
+   single: Bundle; Dependencies
+
+How to Use Bundle Dependencies to load other Bundles
+====================================================
+
+When working on your own bundle(s), you'll sometimes have the need to reuse
+other bundles, either by just requiring, overriding or inherit from them.
+
+While Composer takes care about letting you make sure these dependencies are
+loaded, you'll also need to enable these bundles in the Kernel.
+
+If your bundle is meant for reuse, then this will typically complicate your
+installation documentation, and if your dependencies change over time they
+will likewise complicate your upgrade documentation, and make upgrading
+more tedious for your users.
+
+Starting with Symfony 2.7 you can avoid this by specifying your dependencies
+by implementing "BundleDependenciesInterface" interface::
+
+    // src/Acme/UserBundle/AcmeCacheBundle.php
+    namespace Acme\CacheBundle;
+
+    use Symfony\Component\HttpKernel\Bundle\Bundle;
+
+    class AcmeCacheBundle extends Bundle implements BundleDependenciesInterface
+    {
+        public function getBundleDependencies()
+        {
+            return array('FOS\HttpCacheBundle\FOSHttpCacheBundle');
+        }
+    }
+
+If your bundle requires PHP 5.5 or higher you can also take advantage of the "CLASS"
+constant to be more explicit::
+
+    // src/Acme/UserBundle/AcmeFileBundle.php
+    namespace Acme\FileBundle;
+
+    use Symfony\Component\HttpKernel\Bundle\Bundle;
+
+    class AcmeFileBundle extends Bundle implements BundleDependenciesInterface
+    {
+        public function getBundleDependencies()
+        {
+            return array(Oneup\FlysystemBundle\OneupFlysystemBundle::CLASS);
```

Comment:
What about the way it was done in the api doc?:

```
     * Example of use:
     * ```php
     * class AcmeBundle extends Bundle implements BundleDependenciesInterface
     * {
     *     public function getBundleDependencies()
     *     {
     *         return array(
     *
     *             // All values must be FQN strings to avoid bundles being loaded several times
     *             'FOS\HttpCacheBundle\FOSHttpCacheBundle',
     *
     *             // If you require PHP 5.5 or higher it is better to use `::CLASS` constant:
     *             Oneup\FlysystemBundle\OneupFlysystemBundle::CLASS
     *         );
     *     }
     * }
     * ```
```